bundles/org.openhab.binding.meater/src/main/resources/OH-INF/i18n/meater_xx.properties

```diff
@@ -0,0 +1,21 @@
+# FIXME: please substitute the xx with a proper locale, ie. de
+# FIXME: please do not add the file to the repo if you add or change no content
```

Comment:
This file has to be removed.
And you have to generate the default translations file by running this command: `mvn i18n:generate-default-translations`

Reply:
Fixed!